nixos/modules/services/hardware/litra-autotoggle.nix

```
@@ -0,0 +1,73 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.litra-autotoggle;
+
+  natural = with lib.types; addCheck int (x: x >= 0);
+in
+{
+  options = {
+    services.litra-autotoggle = {
+      enable = lib.mkEnableOption "Automatically turn your Logitech Litra device on when your webcam turns on, and off when your webcam turns off";
+
+      serialNumber = lib.mkOption {
+        type = lib.types.nullOr lib.types.str;
+        description = "The serial number of the Logitech Litra device. You can get the serial number using the litra devices command in the litra CLI.";
```

```suggestion
        description = ''
          The serial number of the Logitech Litra device. You can get
          the serial number using the litra devices command in the litra CLI.
        '';
```

Also for other descriptions. There's not a strict limit for this but don't be too long.